src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleShell.cs

```diff
@@ -25,18 +24,6 @@ public static int Start(string bannerText, string helpText, string[] args)
         {
             return Start(null, bannerText, helpText, null, args);
```

Comment:
Do we use first parameter? Can we remove it?

Reply:
The first param is RunspaceConfiguration.  It's being removed as part of https://github.com/PowerShell/PowerShell/pull/4942

Reply:
Thanks! 

Closed.